src/commands/addons/admin/open.ts

```diff
@@ -1,45 +1,40 @@
-/* tslint:disable */
-// CommandExtension
-import CommandExtension from '../../../CommandExtension';
+import cli from 'cli-ux'
 
-// other packages
-import cli from 'cli-ux';
+import AdminBase from '../../../admin-base'
+import {ReadManifest} from '../../../manifest'
 
-// utilities
-import { readManifest } from '../../../utils/manifest';
-
-export default class Open extends CommandExtension {
+export default class Open extends AdminBase {
   static description = 'open add-on dashboard'
 
-  static args = [{name: 'slug',  description: 'slug name of add-on'}];
+  static args = [{name: 'slug', description: 'slug name of add-on'}]
 
   static examples = [
-`$ heroku addons:admin:open
+    `$ heroku addons:admin:open
 Checking addon_manifest.json... done
-Opening https://addons-next.heroku.com/addons/testing-123... done`, ];
-
+Opening https://addons-next.heroku.com/addons/testing-123... done`,
```

Comment:
We're got a hardcoded fake URL here -- can you interpolate the real URL it will open?